### PR TITLE
feature/google fix

### PR DESCRIPTION
**Pull request by mastertheblaster**
Allow selecting the option from google autocomplete list.

### CODE REVIEW

File: src/GoogleAddressInput/GoogleAddressInput.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ class GoogleAddressInput extends React.Component {
           onKeyDown={this.onKeyDown}
           onBlur={this.onBlur}
           onFocus={this.onFocus}
-          onSelect={this.onSet}
+          onSelect={option => this.onSet(option.value)}
```
Comment (mastertheblaster) [inline src/GoogleAddressInput/GoogleAddressInput.js]: @nirhart `onSelect` passes as the `option` object `{id: '', value: ''}`

File: src/GoogleAddressInput/GoogleAddressInput.spec.js [hunk 2]
```diff
@@ -25,9 +25,13 @@ class GmapsTestClient {
...
-    return Promise.resolve(
-      [_.extend({}, GEOCODE_RESULT, {__called__: JSON.stringify(request)})]
-    );
+    const {address, placeId} = request;
+    if (address || placeId) {
+      return Promise.resolve(
+        [_.extend({}, GEOCODE_RESULT, {__called__: JSON.stringify(request)})]
+      );
+    }
+    throw new Error('geocode() request params are malformed');
```
Comment (mastertheblaster) [inline src/GoogleAddressInput/GoogleAddressInput.spec.js]: @nirhart Made sure the GoogleTestClient fails if wrong arguments are passed.

File: src/GoogleAddressInput/GoogleAddressInput.js (hunk 1 above)
Comment (nirhart) [inline src/GoogleAddressInput/GoogleAddressInput.js]: @3LOK do you want that change?

Comment (3LOK) [inline src/GoogleAddressInput/GoogleAddressInput.js]: @mastertheblaster @nirhart I'm not sure if this change is reflected to the outside. I think the API has remained unchanged. Am I mistaken?

Comment (mastertheblaster) [inline src/GoogleAddressInput/GoogleAddressInput.js]: Nope, API has left exactly the same.

Comment (3LOK) [inline src/GoogleAddressInput/GoogleAddressInput.js]: Ok, so no issues on my side.

**Pull request merged**